pr_agent/settings/configuration.toml

```diff
@@ -1,15 +1,15 @@
 [config]
-model="gpt-4"
-fallback_models=["gpt-3.5-turbo-16k"]
+model="gpt-4-1106-preview"
+fallback_models=["gpt-4"]
 git_provider="github"
 publish_output=true
 publish_output_progress=true
 verbosity_level=0 # 0,1,2
 use_extra_bad_extensions=false
 use_repo_settings_file=true
 ai_timeout=180
-max_description_tokens = 500
-max_commits_tokens = 500
+max_description_tokens = 1000
```

Comment:
Increase max_description_tokens and max_commits_tokens because gpt-4 turbo model is 3x and 2x cheaper respectively compared to the previous GPT-4 pricing.